content/src/content/jcr_root/apps/core/wcm/sandbox/commons/ui/clientlibs/panelcontainer/js/PanelContainer.js

```diff
@@ -20,6 +20,7 @@
     var GET_DATA_SUFFIX = ".model.json";
     var POST_SUFFIX = ".childreneditor.html";
     var MESSAGE_ID = "cmp.panelcontainer";
+    var NAVIGATE_DELAY = 200;
```

Comment:
@jckautzmann, @vladbailescu - this timeout is a little brittle, it's an approximation on the time to update/navigate the UI component after messaging the content frame. Without receiving a return message I don't see any other approach. Unless you have an idea?